### ISSUE TITLE
Missing win64 libraries and build configuration

### ISSUE THREAD

**Issue by yuxuanchiadm** [McKay42/McEngine#20]
There is no win64 libraries for linker to link when building x64 build lol. To build x64 version of engine. We need both add x64 libraries, x64 build configuration and test them.

**Comment by McKay42** [comment]
Yes, I know that. Only 32 bit builds are supported on Windows for now. This is something I'll only do when I really need it (feel free to add them yourself in the meantime though).
